React/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,44 @@
+import { isArray } from "./utils";
+
+export function min(...args) {
+  if (args.length == 0) {
+    return undefined;
+  }
+
+  if (isArray(...args)) {
+    var num = Math.min(...args[0]);
+    return num;
+  }
+
+  return Math.min(...args);
+}
+
+export function copy(args) {
+  var example = args;
+
+  if (isArray(example)) {
+    var copieExample = [...example];
+    return [...example], copieExample;
+  }
+
+  return { ...args };
+}
+
+export function reverseMerge(...args) {
+  let final = [];
+  args.forEach(arg => {
```

Comment:
Have you tried the `.reverse()` method? MDN has some [nice examples](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse) you can try! 😺